app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -408,21 +429,23 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
             Tangram.addDataSource(searchResults);
         }
 
-        var activeFeature: Int = 0
+        var featureCount: Int = 0
         searchResults?.clear()
         for (feature in features) {
             val simpleFeature = SimpleFeature.fromFeature(feature)
             val lngLat = LngLat(simpleFeature.lng(), simpleFeature.lat())
             val properties = com.mapzen.tangram.Properties()
+            properties.add("tap", "search")
```

Comment:
I think we can get rid of "type", "point" property here. Its added to the demo tangram app as an example usage of the property.

With respect to `"tap"`, `"search"` property name and value, these are specific values set in the app (in accordance with the usage in the style sheet). I don't think we need to add these to tangram library. 

Is there a specific file/styling convention you are following for constants, let me know and I can add these.